### DISCUSSION TITLE
HAL dev ref doc status

### DISCUSSION THREAD

**Discussion by holla2040** [grblHAL/core#25] [General]
Hardware Abstraction Layer (HAL) developer reference
https://github.com/grblHAL/core/wiki/Hardware-Abstraction-Layer-(HAL)----developer-reference

This reference was recently edited but mentions 2019-04-12 at the end. I know move to grblHAL is a tremendous effect and this doc might be old. How current is this? I'm now working on a port to Teknic ClearCore hardware. 
Thanks.

**Comment by terjeio** [comment]
Updating the documentation is next on my todo list. My intention is to use the [STM32F7xx driver](https://github.com/grblHAL/STM32F7xx) as reference implementation and use that for bringing documentation up to speed.

**Reply by holla2040** [reply]
Thanks for the quick update, I was looking at the D21 port, will switch to F7xxx driver now. Looking forward to the HAL ref doc update but I understand you're probably really busy. Your answer here is enough to keep working.
